src/index.ts

```diff
@@ -1,4 +1,13 @@
-import { FastCache } from './FastCache';
+import { FastCache, FastCacheOpts, ListOperations, MapOperations, SetOperations } from './FastCache';
 import { InMemoryCache } from './InMemoryCache';
 
-export { InMemoryCache, FastCache, FastCache as RedisCache };
+export {
+  FastCache,
+  FastCache as RedisCache,
+  FastCacheOpts,
+  FastCacheOpts as RedisCacheOpts,
+  InMemoryCache,
+  ListOperations,
+  MapOperations,
+  SetOperations,
+};
```

Comment:
뭐하러 굳이 import / export를 따로......
```suggestion
export {
  FastCache,
  FastCache as RedisCache,
  FastCacheOpts,
  FastCacheOpts as RedisCacheOpts,
  ListOperations,
  MapOperations,
  SetOperations,
} from './FastCache';
export { InMemoryCache } from './InMemoryCache';
```

Reply:
넵 수정하겠습니다!